src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -625,6 +627,12 @@ def forward(
         output_hidden_states=None,
         return_dict=None,
     ) -> Union[Tuple[torch.Tensor], BaseModelOutputWithPastAndCrossAttentions]:
+        if position_ids is not None:
+            warnings.warn(
+                "`position_ids` is deprecated and will be removed in v5.0.0",
```

Comment:
Thought `position_ids` were simply wrong, or are there use cases where people might have passed `position_ids` to overwrite alibi? But actually looking a bit closer into the code, it seems like `position_ids` is never used, so I think we can safely remove it and should therefore adapt this message to "`position_ids` have no functionality in BLOOM and will be removed in v5.0.0. You can safely ignore passing `position_ids`."